### ISSUE TITLE
problem compiling cras  when installing a chroot

### ISSUE THREAD

**Issue by ducksosu** [dnschneid/crouton#4956]
```
Please paste the output of the following command here: sudo edit-chroot -all
```
name: jammy
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/jammy...
crouton: version 1-20221126232257~master:ae594340
release: jammy
architecture: amd64
targets: xfce,core,audio
host: version 15329.52.0 (Official Build) stable-channel octopus 
kernel: Linux localhost 4.14.254-20279-gdb10dc132e24 #1 SMP PREEMPT Wed Mar 15 19:27:16 PDT 2023 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/jammy...
#### Please describe your issue:
cant get the installer to install 
when compiling cras it gives me this:
`Compiling CRAS (i386)...
gcc: error: unrecognized command-line option '-Wundef-prefix=HAVE_,CRAS_'
Failed to complete chroot setup.
Unmounting /mnt/stateful_partition/crouton/chroots/jammy...`
#### If known, describe the steps to reproduce the issue:
sudo crouton -r jammy -t xfce,core,audio

**Comment by CroutonIsFun** [comment]
Agreed. Why can't they remove cras? In my experience, it is not needed.

**Comment by CroutonIsFun** [comment]
Are the other two already installed?

**Comment by DennisLfromGA** [comment]
See: Figure out a way to build audio target now that autotools is removed #4958
